src/files.js

```diff
@@ -246,7 +246,10 @@ FOAM_FILES([
   { name: "foam/classloader/WebModelFileDAO", flags: ['web', 'debug'] },
   { name: "foam/classloader/NodeModelFileDAO", flags: ['node', 'debug'] },
   { name: "foam/classloader/NodeJsModelExecutor", flags: ['node', 'debug'] },
-  { name: "foam/parsers/html" },
-  { name: "foam/parsers/HTMLLexer" },
+  { name: "foam/parsers/html/Attribute" },
```

Comment:
I think proper deps order should demands Attribute and TagType before Tag; Tag before HTMLLexer. Maybe put lib first, too?